### ISSUE TITLE
Raw endpoints unable to capture route params

### ISSUE THREAD

**Issue by tuckers-tech** [encoredev/encore#1223]
Hey Encore team!  I came across an issue today where I need some guidance on a path forward.  When using the TS raw endpoints on a route that is static (no params) it works wonderfully.  When adding an addition of a route param ([id as an example here](https://github.com/tuckers-tech/raw-endpoint-slug-issue/blob/main/url/url.ts#L17)) I am unable to run encore with the following error.
```
$ encore run

  ❌ Building Encore application graph... Failed: parse path param types
Caused by:
    path param "id" not found
  ⠙ Analyzing service topology...
parse path param types

Caused by:
    path param "id" not found
```

I tracked the `RawHandler` type to [here](https://github.com/encoredev/encore/blob/ec3f5f64cbde275915e2c91397db961d56301514/runtimes/js/encore.dev/api/mod.ts#L56) thinking there might be a place for me to add the params in but I don't see any there.

[Minimal reproduction](https://github.com/tuckers-tech/raw-endpoint-slug-issue/blob/main/url/url.ts)

Are there any paths I haven't explored here?

**Comment by eandre** [comment]
Thanks for the report! I've fixed this in #1227. Will issue a new release shortly. (Once live, you can access the path params via `currentRequest()` imported from the `encore.dev` module.)

**Comment by eandre** [comment]
(The fix is now live, with Encore v1.37.7.)
